### PR TITLE
changes for dynamic fluentbit openssl

### PR DESCRIPTION
**Pull request by hestolz**

### CODE REVIEW

**Review by thperapp — APPROVED** [review]

File: AzureMonitorAgent/agent.py [hunk 1]
```diff
@@ -311,6 +313,9 @@ def copy_mdsd_binaries():
     if canUseSharedmdsd != 0 and canUseSharedmdsdmgr != 0:
```
Comment (thperapp) [inline AzureMonitorAgent/agent.py:313]: should we add a fluenbit check?

Comment (thperapp) [reply AzureMonitorAgent/agent.py:313]: resolved offline. There is already an openssl version check

**Review by thperapp — APPROVED** [review]

**Review by simathih — APPROVED** [review]

**Comment by simathih** [comment]
are we making sure fluentbit is being built using dynamic openssl in the dynamic ssl deb/rpm?

**Comment by hestolz** [comment]
> are we making sure fluentbit is being built using dynamic openssl in the dynamic ssl deb/rpm?

I'm testing changes in Compute-Runtime-Tux to allow for selection of static/dynamic openssl for fluent-bit. Will merge this once those are confirmed good.

**Pull request merged**